integration-libs/punchout/root/model/punchout.model.ts

```diff
@@ -24,7 +25,6 @@ export enum PunchOutOperation {
   CREATE = 'CREATE',
   EDIT = 'EDIT',
   INSPECT = 'INSPECT',
-  SOURCE = 'SOURCE',
```

Comment:
[QUESTION]
Why we removed `SOURCE` punchout operation type in this PR? Was it a direct result of adding an Angular Guard? or it was needed anyway to be added, and we just removed it in this PR by the way?

Reply:
Accelerator did not support this operation, neither us.
https://github.tools.sap/cx-commerce/subscriptionbundles/blob/develop/b2bpunchout/resources/b2bpunchout-beans.xml#L22
```
<enum class="de.hybris.platform.b2b.punchout.constants.PunchOutSetupOperation">
		<description>Supported PunchOut operations</description>
		<value>CREATE</value>
		<value>EDIT</value>
		<value>INSPECT</value>
	</enum>
```
	It is used to initiate a quote with a sourcing application. More info on
	https://xml.cxml.org/current/cXMLReferenceGuide.pdf page 79

Reply:
i removed it as it was preventing me to create below type:
export type PunchoutNavigationGuardConfig = {
  [key in PunchOutOperation]: {
    urls?: string[];
    cxRoutes?: string[];
    homePage?: boolean;
  };
};